views/layouts/helpers.js

```diff
@@ -0,0 +1,8 @@
+export const headerTitle = (pathname) => {
+  switch (true) {
+    case pathname === '/configs':
```

Comment:
can we use a route constant here?

Reply:
[c230a13](https://github.com/AMP-SCZ/dpdash/pull/317/commits/c230a139bd58461bc30da48a7f29c3446f7970d3)